### ISSUE TITLE
NavigationControl not working

### ISSUE THREAD

**Issue by VictorBaudot** [uber/react-map-gl#805]
Zoom buttons appear but when I click on them, nothing happens. 
I have the last version of the library.

```javascript
import React, { Component } from 'react';
import MapGL, {
  Marker,
  Popup,
  NavigationControl,
  FullscreenControl
} from 'react-map-gl';

const fullscreenControlStyle = {
  position: 'absolute',
  top: 0,
  left: 0,
  padding: '10px'
};

const navStyle = {
  position: 'absolute',
  top: 36,
  left: 0,
  padding: '10px'
};

class App extends Component {
  constructor(props) {
    super(props);
    this.state = {
      viewport: {
        width: '100%',
        height: '100%',
        latitude: 37.7577,
        longitude: -122.4376,
        zoom: 3.5,
        bearing: 0,
        pitch: 0
      }
    };
  }

  render() {
    const { viewport } = this.state;
    return (
      <MapGL
        {...viewport}
        mapStyle="mapbox://styles/mapbox/dark-v9"
        onViewportChange={viewport => this.setState({ viewport })}
      >
        <div className="fullscreen" style={fullscreenControlStyle}>
          <FullscreenControl />
        </div>
        <div className="nav" style={navStyle}>
          <NavigationControl />
        </div>
      </MapGL>
    );
  }
}

export default App;
```

**Comment by Pessimistress** [comment]
You need to follow the documentation: http://uber.github.io/react-map-gl/#/Documentation/api-reference/navigation-control and provide a `onViewportChange` callback.

The latest release is published from the 4.1-release branch. We recently landed a change that makes this prop optional onto master, but it is not published yet.

**Comment by Pessimistress** [comment]
The above feature is published in 4.1.4.

**Comment by VictorBaudot** [comment]
@Pessimistress Thank you, just upgraded to 4.1.4 and it's working :)
